style.css

```diff
@@ -4,7 +4,7 @@
     font-family:'poppins', sans-serif;
 }
     #banner{
-        background: linear-gradient(rgba(0,0,0,0.5),#009688), url(./Barber_Shop_img/banner.jpg);
+      background-color: antiquewhite;
```

Comment:
this is wrong, it should be a background image.

Reply:
GOOD JOB